registry/storage/driver/middleware/rewrite/middleware.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func init() {
 	if err := storagemiddleware.Register("rewrite", newRewriteStorageMiddleware); err != nil {
-		logrus.Errorf("failed to register redirect storage middleware: %v", err)
+		logrus.Errorf("failed to register rewrite storage middleware: %v", err)
```

Comment:
Unrelated to this PR; not sure what we did in the past; even curious if failing to register things should be a hard failure (panic / fatalf)